### PR TITLE
add auth callback page

### PR DESCRIPTION
**Pull request by tebogo-bbd**
This will allow the backend to communicate with the frontend
after a user successfully authenticates with github the api generates a jwt and the frontend will save it

### CODE REVIEW

File: frontend/callback.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Authentication Callback</title>
+    <style>
```
Comment (Vutlhari-Mashimbyi) [inline frontend/callback.html]: should move the styles to the css file

File: frontend/callback.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        #message {
+            color: #0a0707;
+            font-size: 1.2em;
+            margin: 20px;
```
Comment (Vutlhari-Mashimbyi) [inline frontend/callback.html]: I heard that we must not use px's 😅

File: frontend/callback.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            padding: 10px;
+            background-color: #e2e8f0; 
+            text-align: center;
+        }
+    </style>
+</head>
+<body>
+    <div id="message">Loading...</div> <!-- Placeholder for messages -->
```
Comment (Vutlhari-Mashimbyi) [inline frontend/callback.html]: Apparently we never use a div 😅. Maybe a <p> tag?

File: frontend/callback.html (hunk 1 above)
Comment (tebogo-bbd) [reply frontend/callback.html]: I dont want to import the whole css file since it's large 🤣

File: frontend/callback.html (hunk 2 above)
Comment (tebogo-bbd) [reply frontend/callback.html]: ohhhh yha I'll make a plan

File: frontend/callback.html (hunk 3 above)
Comment (tebogo-bbd) [reply frontend/callback.html]: eish alright 🤣

File: frontend/callback.html (hunk 1 above)
Comment (Vutlhari-Mashimbyi) [reply frontend/callback.html]: I see😅

File: frontend/callback.html (hunk 3 above)
Comment (Vutlhari-Mashimbyi) [reply frontend/callback.html]: We don't wanna pick fights here and there🤣

**Review by Vutlhari-Mashimbyi — APPROVED** [review]

**Pull request merged**